txnkv/transaction/2pc.go

```diff
@@ -82,7 +83,7 @@ var (
 
 var (
 	// PrewriteMaxBackoff is max sleep time of the `pre-write` command.
-	PrewriteMaxBackoff = 40000
+	PrewriteMaxBackoff = atomicutil.NewUint32(40000)
```

Comment:
`CommitMaxBackoff` is a uint64. What about also use uint64 here to be more consistent?

Reply:
OK, have updated.